js/index.d.ts

```diff
@@ -1,47 +1,159 @@
-/*
-  Typings for @sentry/cli
-*/
+/**
+ * Typings for @sentry/cli
+ */
 declare module '@sentry/cli' {
   export interface SentryCliOptions {
+    /**
+     * Sentry instance
```

Comment:
This may be a bit brittle, but can we reference the env variables https://github.com/getsentry/sentry-cli/blob/49eeb9aab399cb92b097c3a6c213c9c8dcf0c0df/js/helper.js#L142 that the config options update? That way people can also search up the env variables and get see associated docs/GH issues.